content/docs/app-developer-guide/build-an-arm-app.md

```diff
@@ -0,0 +1,92 @@
++++
+title="Build an ARM app"
+weight=2
+summary="The basics of taking your app from source code to runnable ARM image."
++++
+
+As of today, there are no CNB platforms (e.g. [pack][pack]) that support building ARM application images. In the following tutorial, we will be performing a build "manually", in that we will be performing a build by invoking the lifecycle directly.
+
+### 1. Prepare your working directory
+
+On your Linux ARM machine with a [docker][docker] daemon installed, prepare the following directory tree structure.
+
+```bash
+tree ~/workspace/
+~/workspace/
+├── buildpacks
+│   └── hello-world
+└── platform
+```
```

Comment:
I think that I would put it after the cloning part, and then you can show the full `tree` (how the `workspace` directory supposed to look like).

Reply:
If I showed the full samples repo in the diagram, the diagram would get really big and quickly get out of date... 🤷🏾‍♂️